src/main/java/online/shop/model/Product.java

```diff
@@ -0,0 +1,41 @@
+package online.shop.model;
+
+public class Product {
+    private Long id;
+    private String name;
+    private Double price;
```

Comment:
```suggestion
    private double price;
```
this is good practice to use a wrapper for id
here we can use a primitive type